examples/char/.gitignore

```diff
@@ -0,0 +1,4 @@
+package-lock.json
+char.js
+char_bg.js
+char_bg.wasm
```

Comment:
Could this ignore `dist` as well? (I think a 1.2MB js file was checked in below)